### ISSUE TITLE
passwords app keeps reloading and messes up session

### ISSUE THREAD

**Issue by mathroc** [marius-wieschollek/passwords#55]
### System Information
- Nextcloud Version: 13.0.4
- PHP Version: 7.1.18
- Passwords Version:  2018.6.3
- Browser and Version: 63.0a1 (2018-06-26)
- Client OS and Version: Gentoo
- Server OS and Version: ubuntu / docker 18.03.1

### Steps to reproduce
1. Install the passwords app
2. go to the passwords app

### Actual result

An empty passwords folder appears (that part's normal)
A tooltip appears at the top of the page :

> Problem loading page, reloading in 5 seconds

After a few reload I'm logged out

In the browser console I can see a few 401 errors on those urls:

- /apps/encryption/ajax/getStatus
- /apps/files/
- /apps/apporder/getOrder
- /ocs/v2.php/apps/notifications/api/v2/notifications

if I go fast I can still add a password or view existing ones

this also cause some authorization problems, after that I cannot see my files (needs to logout and login again)

### Expected result

no reloading, and no weird stuff :)

**Comment by marius-wieschollek** [comment]
I just tried with Firefox 63.0a1 on my Ubuntu 18.04 and saw the same issue. Firefox Nightly sends the session cookie to the api despite the default setting for fetch being _not_ to send cookies.

Since the stable Firefox 60 works fine i assume that this is an issue with the credentials option of the fetch request in Firefox Nightly. I have extended the configuration for the api request to explicitly exclude cookies and the nightly build 2018.7.0-build2334 will ship the fix.
